### PR TITLE
Implement an exporter for target message counts

### PR DESCRIPTION
**Pull request by torotil**

### CODE REVIEW

File: campaignion_csv.module [hunk 1]
```diff
@@ -112,6 +113,14 @@ function campaignion_csv_campaignion_csv_info() {
...
     ],
   ];
+  $export['target_message_count'] = [
+    'file_pattern' => [
+      'class' => YearlyFilePattern::class,
+      'path' => 'targets/%Y.csv',
+      'retention_period' => new \DateInterval('P24M'),
+      'refresh_interval' => new \DateInterval('PT23H30M'),
+    ],
+  ];
```
Comment (katzmo) [inline campaignion_csv.module:129]: Doesn’t this need exporter settings too? How does the export know to use the TargetStatsExporter?

File: src/Exporter/TargetStatsExporter.php [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  INNER JOIN webform_component c USING(nid)
+  INNER JOIN webform_submitted_data d USING(nid, sid, cid)
+WHERE c.type='e2t_selector' AND s.submitted BETWEEN :start AND :end AND s.sid>:last_sid
+LIMIT 10000
+SQL;
+    $last_sid = 0;
+    $args = [':start' => $start, ':end' => $end - 1];
+    $message_counts = [];
+    $targets = [];
+    $continue = TRUE;
```
Comment (katzmo) [inline src/Exporter/TargetStatsExporter.php]: Look like the variable is not used anywhere, maybe it was left-over?

File: campaignion_csv.module (hunk 1 above)
Comment (torotil) [reply campaignion_csv.module:129]: Yes, that’s  fixed in the newest version.

File: src/Exporter/TargetStatsExporter.php (hunk 2 above)
Comment (torotil) [reply src/Exporter/TargetStatsExporter.php]: Fixed in the latest version.

I also removed `$message_counts` for the same reason.

**Review by katzmo — APPROVED** [review]

**Pull request merged**